sdk/core/abort-controller/package.json

```diff
@@ -61,38 +76,32 @@
     "browser",
     "cloud"
   ],
-  "author": "Microsoft Corporation",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
-  },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/abort-controller/README.md",
-  "sideEffects": false,
   "dependencies": {
     "tslib": "^2.2.0"
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@microsoft/api-extractor": "^7.31.1",
-    "@types/chai": "^4.1.6",
+    "@types/chai": "^4.3.5",
```

Comment:
can we avoid bumping dependencies in this PR? If it's required we should do it in a separate PR and merge through main